Nef_3/include/CGAL/Nef_3/SNC_io_parser.h

```diff
@@ -1579,7 +1579,7 @@ read_vertex(Vertex_handle vh) {
   vh->sncp() = this->sncp();
 
   in >> index;
-  if(index >= int(en))
+  if(index >= int((int)(en)))
```

Comment:
isn't this a double cast? could be written as:
`(int)en`, instead of `int((int)en)`?